model/src/main/java/com/javahelp/model/user/User.java

```diff
@@ -0,0 +1,80 @@
+package com.javahelp.model.user;
+
+/**
+ * A class for a user entity
+ */
+public class User {
+
+    /**
+     * The ID of the user account (randomly generated and immutable)
+     */
+    private final String stringID;
```

Comment:
this shouldn't be final, according to our spec right now, it needs to be reassigned by the frontend when fetching a newly created user.

Reply:
It also needs a setter, I think it already has a getter, but it needs a setter too (the setter just won't be used very often)